src/components/textarea/textarea.component.js

```diff
@@ -72,13 +72,12 @@ const Textarea = ({
 
   const inputRef = useRef();
 
-  const minHeight = useRef(0);
+  const minHeight = useRef(40);
 
   const expandTextarea = () => {
     const textarea = inputRef.current;
 
     if (textarea.scrollHeight > minHeight.current) {
-      // Reset height to zero - IE specific
```

Comment:
This is necessary for the expandable prop to work correctly, not IE specific at all.